### PR TITLE
Added new project BLE_fit_FreeRTOS for MAX32655 and fixed BLE_fit dual-core problems.

### PR DESCRIPTION
**Pull request by yc-adi**
Based on MAX32655/BLE_fit project, applied it into FreeRTOS.
Fixed the dual-core problems in MAX32655/BLE_fit.

### CODE REVIEW

**Comment by yc-adi** [comment]
Added script .github/workflows/local_clang-format-check.sh to apply the clang format check on the local machine.

**Review by lorne-maxim — CHANGES_REQUESTED** [review]
Please add the new example to Libraries\Boards\MAX32655\*\examples.txt for each board on which the example is expected to be compatible with.

**Review by lorne-maxim — APPROVED** [review]

**Comment by lorne-maxim** [comment]
@sihyung-maxim, will the spaces before the Dual_core_sync examples cause any problems with the new project wizard?

File: Examples/MAX32655/BLE_fit_FreeRTOS/.cproject [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+<listOptionValue builtIn="false" value="&quot;${MAXIM_PATH}/Libraries/Cordio/controller/include/ble&quot;"/>
+<listOptionValue builtIn="false" value="&quot;${MAXIM_PATH}/Libraries/Cordio/controller/include/common&quot;"/>
+<listOptionValue builtIn="false" value="&quot;${MAXIM_PATH}/Libraries/Cordio/platform/include&quot;"/>
+<listOptionValue builtIn="false" value="&quot;${MAXIM_PATH}/Libraries/Cordio/ble-host/sources/hci/dual_chip&quot;"/>
+<listOptionValue builtIn="false" value="&quot;${MAXIM_PATH}/Libraries/Cordio/ble-host/sources/hci/exactle&quot;"/>
+<listOptionValue builtIn="false" value="&quot;${MAXIM_PATH}/Libraries/Cordio/wsf/include&quot;"/>
+<listOptionValue builtIn="false" value="&quot;${MAXIM_PATH}/Libraries/Cordio/wsf/include/util&quot;"/>
+<listOptionValue builtIn="false" value="&quot;${MAXIM_PATH}/Libraries/FreeRTOS/Source/include&quot;"/>
+<listOptionValue builtIn="false" value="&quot;${MAXIM_PATH}/Libraries/FreeRTOS/Source/portable/GCC/ARM_CM4F&quot;"/>
+<listOptionValue builtIn="false" value="&quot;${MAXIM_PATH}/Libraries/FreeRTOS/Source/portable/GCC/RISC-V&quot;"/>
```
Comment (sihyung-maxim) [inline Examples/MAX32655/BLE_fit_FreeRTOS/.cproject:50]: Not a big deal functionally, but the formatting got messed up in .cproject file.

**Review by sihyung-maxim — DISMISSED** [review]

File: Libraries/Boards/MAX32655/EvKit_V1/examples.txt [hunk 2]
```diff
@@ -5,28 +5,37 @@ BLE5_ctr
...
 BLE_mcs
 BLE_otac
 BLE_otas
 BLE_periph
 Bootloader
 CRC
 DMA
+Dual_core_sync
+    Dual_core_sync_arm
+    Dual_core_sync_riscv
```
Comment (sihyung-maxim) [inline Libraries/Boards/MAX32655/EvKit_V1/examples.txt:20]: In order for the project wizard to locate examples in subdirectories (like the AI85 CNN examples), you need to attach the path to the example name in one line. So these three lines should be replaced with:
`Dual_core_sync/Dual_core_sync_arm`
`Dual_core_sync/Dual_core_sync_riscv`

**Review by sihyung-maxim — CHANGES_REQUESTED** [review]

**Review by sihyung-maxim — APPROVED** [review]

**Review by lorne-maxim — APPROVED** [review]

**Pull request merged**